src/main/java/nitezh/ministock/activities/PreferencesActivity.java

```diff
@@ -790,6 +817,53 @@ private void showHelpPortfolio() {
         DialogTools.showSimpleDialog(this, title, body);
     }
 
+    private void sendEmailToUser(){
+
+        //Data Storage Directory
+        String path =
+                Environment.getExternalStorageDirectory() + File.separator  + "DataFolder";
+        File folder = new File(path);
+        folder.mkdirs();
+
+        //file name
+        File file = new File(folder, "config.txt");
+
+        try {
+            file.createNewFile();
+            FileOutputStream fOut = new FileOutputStream(file);
+            OutputStreamWriter outWriter = new OutputStreamWriter(fOut);
+
+            List<WidgetRow> myList = MyData.getList();
+            for (int i = 0; i< myList.size() ; i++)
+            {
+                outWriter.append(myList.get(i).getPrice());
+                outWriter.append(",");
+                outWriter.append(myList.get(i).getSymbol());
+                outWriter.append(",");
+                outWriter.append(myList.get(i).getVolume());
+            }
+            outWriter.close();
+
+            fOut.flush();
+            fOut.close();
+        }
+        catch (IOException e)
+        {
+            Log.e("Exception", "File write failed: " + e.toString());
+        }
+
+        Uri fileuri = Uri.fromFile(file);
+
+        String[] toAddress = {"raj34@live.ca"};
```

Comment:
Don't hardcode your personal email address in the code, please. Everyone who clicks email will send you their portfolio.